README.md

```diff
@@ -130,6 +131,12 @@ Flow2ML for Python can be installed from Source,Pypi and Docker container instal
     # Obtain Train, Validation data splits
     (train_x, train_y, val_x, val_y) = flow.getDataset( img_dimensions, test_val_split )
 
+    # Define The conversions to be used
+    model={'tfjs':False, 'tflite':False}
```

Comment:
Either tfjs or tflite needs to be true and don't use model as a variable as it would confuse the user.